### PR TITLE
added update to full CIL

### PR DESCRIPTION
**Pull request by paskino**

### CODE REVIEW

File: scripts/update_VM_to_full_CIL.sh [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+# first do a very brief check
+if [ -z "$SIRF_PATH" -o ! -d $SIRF_PATH -o ! -d $SIRF_PATH/../STIR ]; then
+   echo "Directories not found. Run update_VM.sh"
+   exit 1
+fi
+
+# change build files to also build the STIR executables
+cd $SIRF_PATH/../..
+cmake -DBUILD_CIL:BOOL=ON
```
Comment (KrisThielemans) [inline scripts/update_VM_to_full_CIL.sh:20]:
```suggestion
cmake -DBUILD_CIL:BOOL=ON .
```

File: scripts/update_VM_to_full_CIL.sh [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+make -j2 
+
+# update/get STIR exercises
```
Comment (KrisThielemans) [inline scripts/update_VM_to_full_CIL.sh:23]:
```suggestion
# update/get CIL demos
```

**Review by KrisThielemans — CHANGES_REQUESTED** [review]
I haven't run this (and I don't think you did either)

**Comment by paskino** [comment]
I sincerely thought I had run it, but clearly without that `.` it would have failed.

**Pull request merged**